### PR TITLE
Document changes in the tracker and events API - DHIS2-3396 DHIS2-3226

### PR DESCRIPTION
**Pull request by huzvak**
Document changes in the tracker and events API regarding deleted items and updating and retrieving deleted items - DHIS2-3396 DHIS2-3226

### CODE REVIEW

File: src/docbkx/en/content/developer/web_api.xml [hunk 1]
```diff
@@ -4271,13 +4271,15 @@ FTRrcoaog83,201401,DiszpKrYNg8,bRowv6yZOF2,bRowv6yZOF2,12,system,2014-03-02T21:4
           role="italic">unique</emphasis> metadata attribute value.</para>
       <para><emphasis role="bold">Update</emphasis>: To update an existing event, the format of the
         payload is the same, but the URL you are posting to must add the identifier to the end of
-        the URL string and the request must be PUT.
-        <screen>curl -X PUT -d @updated_event.xml &quot;localhost/api/26/events/ID&quot; 
+        the URL string and the request must be PUT. </para>
+      <para>It is not allowed to update an already deleted event. (The same applies to tracked
+        entity instance and
```
Comment (Bekkalizer) [inline src/docbkx/en/content/developer/web_api.xml]: Id say hunt down the correct passages to add the info for tracked entity instnace and enrollment

File: src/docbkx/en/content/developer/web_api.xml [hunk 2]
```diff
@@ -13042,6 +13044,9 @@ interval=WEEK" -u admin:district</screen>
     }
   ]
 }</screen></para>
+      <para>The system does not allow to create a tracked entity instance (as well as enrollment and
```
Comment (Bekkalizer) [inline src/docbkx/en/content/developer/web_api.xml]: "allow the creation of a"

File: src/docbkx/en/content/developer/web_api.xml [hunk 3]
```diff
@@ -14527,6 +14535,20 @@ interval=WEEK" -u admin:district</screen>
     <screen>curl -X POST -d @data.json -H &quot;Content-Type: application/json&quot;
   &quot;http://server/api/26/events?strategy=DELETE&quot;</screen>
   </section>
+  <section xml:id="webapi_updating_and_deleting_items">
```
Comment (Bekkalizer) [inline src/docbkx/en/content/developer/web_api.xml]: Is there individual sections for event, tei and enrollment where this could have been slotted in?

Comment (huzvak) [reply src/docbkx/en/content/developer/web_api.xml]: That's the thing that there is no section for enrollment at all. There is for Event and for TEI. I put it into both places but also making a new section for it. Ideally, I would reorder the docs and put Events, TEI and Enrollments next to each other as subsections and create a new section called Tracker Web API

**Pull request closed without merging**